graphql-spqr/pom.xml

```diff
@@ -0,0 +1,302 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
```

Comment:
Embedding `graphql-spqr` is because we need some changes upstream that aren't released?

Do they publish SNAPSHOTs anywhere?

Reply:
Yes. There is no SNAPSHOT available. The changes to make it MicroProfile compatible is in a certain branch. This should be released soon, but for now, at least we can continue with creating the MicroProfile TCK

Reply:
I think it depends on the definition of "soon".

Ideally we don't want to have an entire project embedded that's going to be removed if it's just a few weeks.

If the time period is reasonably short, I'd prefer we just have a SNAPSHOT dependency for now. CI won't work, but we wouldn't have huge commits for adding/removing the dependency code.

Do we know what the timeframe might be?

Reply:
I'll find out and get back to you here.

Reply:
Ok so this might still be a while before we see a release for spqr. However, I removed making a copy of all the source, and just added a checkout from the correct branch into target. At least is not committing a bunch of code just to delete it later. Now it's once pom.xml that will be removed later. Let me know if this is better.

Reply:
@kenfinnigan - any feedback ?